pubsub/kafkapubsub/kafka_test.go

```diff
@@ -53,21 +54,21 @@ type harness struct {
 }
 
 var checkOnce sync.Once
-var kafkaRunning bool
+var kafkaError error
```

Comment:
I think the `sync.Once` and `kafkaError` variables and `localKafkaRunning` can all be dropped. The tests will just fail naturally if Kafka isn't running.